cg/cli/demultiplex/demux.py

```diff
@@ -49,7 +47,7 @@ def demultiplex_all(
 
         if not flow_cell.validate_sample_sheet():
             LOG.warning(
-                f"Malformed sample sheet. Run cg demultiplex samplesheet validate {flow_cell.sample_sheet_path}",
+                f"Malformed sample sheet. Run cg demultiplex sample sheet validate {flow_cell.sample_sheet_path}",

```

Comment:
This was initially correct as the command is actually called `cg demultiplex samplesheet validate`. If you want you could add "Run command cg demultiplex sample sheet validate" maybe?

Reply:
reversed the change then